pkg/cache/sql/informer/factory/informer_factory.go

```diff
@@ -20,17 +21,22 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+// EncryptAllEnvVar is set to "true" if users want all types' data blobs to be encrypted in SQLite
+// otherwise only variables in defaultEncryptedResourceTypes have their blobs encrypted
+const EncryptAllEnvVar = "CATTLE_ENCRYPT_CACHE_ALL"
+
 // CacheFactory builds Informer instances and keeps a cache of instances it created
 type CacheFactory struct {
-	informerCreateLock sync.RWMutex
-	wg                 wait.Group
-	dbClient           DBClient
-	stopCh             chan struct{}
-	encryptAll         bool
+	wg          wait.Group
```

Comment:
Doesn't have much to do with this PR but we're never calling `wg.Wait()`. We should probably be calling it after closing the `stopCh` channel... What do you think about fixing this in this PR? Otherwise I can create an issue and leave it for another.

Reply:
fixed in be00af8